### ISSUE TITLE
[FTR] Unblock User

### ISSUE THREAD

**Issue by xbvrhlmb** [austinhuang0131/barinsta#1091]
If you open profile and click Block user
you not have functional Unblock User

need add button "Unblock User"

if prorile refresh you not can open block profile for unblock.

**Comment by austinhuang0131** [comment]
Duplicate of #658 (since visiting a blocked account will actually say the account does not exist in the API)
